packages/devextreme/testing/tests/DevExpress.ui.widgets/listParts/commonTests.js

```diff
@@ -3717,22 +3760,25 @@ QUnit.module('keyboard navigation', {
 
         keyboard.keyDown('up');
         this.clock.tick(10);
-        assert.ok($selectAllCheckBox.hasClass('dx-state-focused'), 'selectAll checkbox is focused');
+        assert.ok($selectAllCheckBox.hasClass(`${LIST_FOCUS_STATE_CLASS}`), 'selectAll checkbox is focused');
 
         keyboard.keyDown('up');
         this.clock.tick(10);
+        const lastItemIsFocused = $lastItem.hasClass(`${LIST_FOCUS_STATE_CLASS}`);
 
-        assert.ok(!$selectAllCheckBox.hasClass('dx-state-focused'), 'selectAll checkbox isn\'t focused');
-        assert.ok($lastItem.hasClass('dx-state-focused'), 'last item is focused');
+        assert.ok(!$selectAllCheckBox.hasClass(`${LIST_FOCUS_STATE_CLASS}`), 'selectAll checkbox isn\'t focused');
+        assert.ok(lastItemIsFocused, 'last item is focused');
 
         keyboard.keyDown('down');
         this.clock.tick(10);
-        assert.ok($selectAllCheckBox.hasClass('dx-state-focused'), 'selectAll checkbox is focused');
+        assert.ok($selectAllCheckBox.hasClass(`${LIST_FOCUS_STATE_CLASS}`), 'selectAll checkbox is focused');
 
         keyboard.keyDown('down');
         this.clock.tick(10);
-        assert.ok(!$selectAllCheckBox.hasClass('dx-state-focused'), 'selectAll checkbox isn\'t focused');
-        assert.ok($firstItem.hasClass('dx-state-focused'), 'first item is focused');
+        const firstItemIsFocused = $firstItem.hasClass(`${LIST_FOCUS_STATE_CLASS}`);
+
+        assert.ok(!$selectAllCheckBox.hasClass(`${LIST_FOCUS_STATE_CLASS}`), 'selectAll checkbox isn\'t focused');
+        assert.ok(firstItemIsFocused, 'first item is focused');
```

Comment:
It looks strange that in this row we used a separate variable but don't do in a row above. Approaches should be consistent everywhere.

For me previous row is simpler to read. I think we should leave this approach.

Reply:
@nikkithelegendarypokemonster  I still see you extract `firstItemIsFocused` variables in some tests